stdr_robot/src/stdr_robot.cpp

```diff
@@ -60,9 +60,13 @@ namespace stdr_robot
     _moveRobotService = n.advertiseService(
       getName() + "/replace", &Robot::moveRobotCallback, this);
 
-    //we should not start the timer, until we hame a motion controller
+    //allow changing odometry rate, as the default 10Hz can be too slow in some cases
+    double odometry_rate;
+    n.param(getName() + "/odometry_rate", odometry_rate, 10.0);
+
+    //we should not start the timer, until we have a motion controller
     _tfTimer = n.createTimer(
-      ros::Duration(0.1), &Robot::publishTransforms, this, false, false);
+      ros::Duration(1.0/odometry_rate), &Robot::publishTransforms, this, false, false);
```

Comment:
Could we put the default to 100Hz and not use the parameter? If you use the parameter you would need to check for division by zero. I also don't know what would be the computational impact if we increase to 100Hz...

Reply:
Impact is nearly zero, I get almost the same CPU usage (around ~85%). What is normal as the function is quite light. But 50 Hz would be equally fine.